.generated/Add.ts

```diff
@@ -0,0 +1,415 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+
+import {
+  web3,
+  Contract as ContractArtifact,
+  SignerProvider,
+  Address,
+  Token,
+  toApiVals,
+  ContractState,
+  node,
+  binToHex,
+  TestContractResult,
+  InputAsset,
+  Asset,
+  HexString,
+  SignDeployContractTxResult,
+  contractIdFromAddress,
+  fromApiArray,
+  ONE_ALPH,
+  groupOfAddress,
+  fromApiVals,
+  subscribeToEvents,
+  SubscribeOptions,
+  Subscription,
+  EventSubscription,
+} from "@alephium/web3";
+
+export namespace Add {
+  export type Fields = {
+    sub: HexString;
+    result: bigint;
+  };
+
+  export type State = Fields & Omit<ContractState, "fields">;
+
+  export type AddEvent = {
+    blockHash: string;
+    txId: string;
+    x: bigint;
+    y: bigint;
+    eventIndex: number;
+  };
+
+  export type Add1Event = {
+    blockHash: string;
+    txId: string;
+    a: bigint;
+    b: bigint;
+    eventIndex: number;
+  };
+
+  export async function deploy(
+    signer: SignerProvider,
+    initFields: Fields,
+    deployParams?: {
+      initialAttoAlphAmount?: bigint;
+      initialTokenAmounts?: Token[];
+      issueTokenAmount?: bigint;
+      gasAmount?: number;
+      gasPrice?: bigint;
+    }
+  ): Promise<SignDeployContractTxResult & { instance: AddInstance }> {
+    const deployResult = await artifact.deploy(signer, {
```

Comment:
We should remove `artifact.deploy` and convert directly from node result, so that there will be only one way to deploy contracts.

We could refactor the conversion code as helper functions in `contract.ts` so we could use them in the generated code.

Reply:
Sorry, I don't understand what you mean by `convert directly from node result`?